app/services/sparqls_count_service.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'lodqa/anchored_pgps'
+require 'lodqa/graph_finder'
+require 'logger/logger'
+require 'sparql_client/cacheable_client'
+
+# returns the count of sparqls
+module SparqlsCountService
+  class << self
+    # Handle sparqls count.
+    def sparqls_count param
+      parallel = 16
+      sparql_client = SparqlClient::CacheableClient.new(param.endpoint_url,
+                                                        parallel, param.endpoint_options)
+      graph_finder = Lodqa::GraphFinder.new(sparql_client,
+                                            param.graph_uri, param.graph_finder_options)
+
+      sparqls_count = 0
+      anchored_pgps = Lodqa::AnchoredPgps.new param.pgp, param.mappings
+      # rubocop:disable Lint/LiteralAsCondition
+      anchored_pgps.logger =
+        Logger::Logger.new 'sparqls_count', nil, false ? Logger::DEBUG : Logger::INFO
```

Comment:
Railsからつかうときは引数loggerにloggerを指定してください。
Railsのログに出力されます。

falseなら常にLogger::INFOになると思います。

```suggestion
      anchored_pgps.logger =
        Logger::Logger.new 'sparqls_count', logger, Logger::INFO
```

で、たぶん動くと思います。

Reply:
`include Logger::Loggable`してから動けました。